.github/workflows/performance.yml

```diff
@@ -13,7 +12,6 @@ on:
   pull_request:
     branches:
       - trunk
-      - 'feature/**'
       - '3.[7-9]'
       - '[4-9].[0-9]'
```

Comment:
See the other review conversation, I think we should remove everything from here for now except lines 4-6, so that this workflow only runs on `push` events for `trunk`.

Covering point releases is a good future enhancement, but we would need to determine first how we can make that work with the Code Health dashboard, as it would probably need multiple instances for those different branches.

Reply:
I agree that we don't want to collect data in the dashboard for now, but I do think it's useful to have these run on feature branches in the mean time so we at least have logs to look back to if a performance improvement/regression gets merged during a backport.

Reply:
@joemcgill I'd argue we should leave that to the discussions on https://core.trac.wordpress.org/ticket/57556 and eventually make a change as part of that ticket.

Reply:
Sorry, mistakenly said "feature branches" when I meant "release branches". I agree that we should remove the feature branches as we've done and defer conversation about feature branches to that Trac ticket.